.golangci.yml

```diff
@@ -97,7 +99,3 @@ linters-settings:
           - github.com/DataDog/datadog-agent/pkg/util/log.Printf
           - github.com/DataDog/datadog-agent/pkg/util/log.Warnf
           - github.com/DataDog/datadog-agent/pkg/util/log.Errorf
-  revive:
-    rules:
-      - name: package-comments
-        disabled: true
```

Comment:
So AFAIUI [these lines](https://github.com/DataDog/datadog-agent/blob/008f08ea19a7692dcfc50f0616f363a7823b1e51/.golangci.yml#L59-L72) are the ones saying golangci should:
- disable all the default linters (`disable-all: true`)
- enable all the linters we specify:
```
  enable:
    - unconvert   # Remove unnecessary type conversions
    - unused      # Checks Go code for unused constants, variables, functions and types
    - ineffassign # Detects when assignments to existing variables are not used
    - misspell    # Finds commonly misspelled English words in comments
    - gofmt       # Gofmt checks whether code was gofmt-ed
    - revive      # Revive is a replacement for golint, a coding style checker
    - errcheck    # errcheck is a program for checking for unchecked errors in go programs.
    - staticcheck # staticcheck is a go vet on steroids, applying a ton of static analysis checks
    - govet       # Vet examines Go source code and reports suspicious constructs, such as Printf calls whose arguments do not align with the format string
    - depguard    # Depguard is useful for preventing specific packages from being used
    - bodyclose 
```
 Since `revive` is part of this list, I guess we're doing fine !

Reply:
Ah, gotcha - that looks right. It seems weird to me to enable and configure the linters in two different places 🤷